package.json

```diff
@@ -1,44 +1,41 @@
 {
-  "name": "app-frontend",
-  "version": "0.1.0",
-  "private": true,
-  "scripts": {
-    "serve": "vue-cli-service serve",
-    "build": "vue-cli-service build",
-    "lint": "vue-cli-service lint",
-    "test:unit": "vue-cli-service test:unit",
-    "format": "prettier --write src/**/*.js"
-  },
-  "husky": {
-    "hooks": {
-      "pre-commit": "npm run lint && npm run test:unit"
+    "name": "app-frontend",
```

Comment:
Please check the indent size. It's overwriting the whole file. It should be 2 instead of 4.
With prettier: `"tabWidth": 2`